client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -28,12 +28,19 @@
 		}
 	}
 
+	.step-container__content {
+		@include break-large {
+			height: 100vh;
```

Comment:
If I understand things correctly, it seems like this is doing the bulk of the heavy lifting with the responsive iframe preview. Is that right?